### ISSUE TITLE
Container doesn't appear to be run-able as non-root despite PR to fix it

### ISSUE THREAD

**Issue by samip5** [jorenn92/Maintainerr#752]
**Describe the bug**
The container seems to still want to run as root despite using 1.7.1 docker tag.

**To Reproduce**
1. Pull `ghcr.io/jorenn92/maintainerr:1.7.1`
2. Change entry point to sh
3. Run cat /etc/supervisord.conf
4. See that it's still set to root.

**Expected behavior**
It should list user as node instead of root.

**Device (please complete the following information):**
 - OS: Debian
 - Version: 1.7.1

**Additional context**
```
docker run --rm -it --entrypoint sh ghcr.io/jorenn92/maintainerr:1.7.1@sha256:7d1e7834c5f9692553fd5515385cb32875e328fd0506965b2afb26e36c6e8e6e
Unable to find image 'ghcr.io/jorenn92/maintainerr:1.7.1@sha256:7d1e7834c5f9692553fd5515385cb32875e328fd0506965b2afb26e36c6e8e6e' locally
ghcr.io/jorenn92/maintainerr@sha256:7d1e7834c5f9692553fd5515385cb32875e328fd0506965b2afb26e36c6e8e6e: Pulling from jorenn92/maintainerr
661ff4d9561e: Pull complete
89059ca18a98: Pull complete
9bb776f8ddb0: Pull complete
ef0b6390b7f1: Pull complete
4f4fb700ef54: Pull complete
e6905e1214a7: Pull complete
63ac157c6cc3: Pull complete
1591770db6ef: Pull complete
Digest: sha256:7d1e7834c5f9692553fd5515385cb32875e328fd0506965b2afb26e36c6e8e6e
Status: Downloaded newer image for ghcr.io/jorenn92/maintainerr@sha256:7d1e7834c5f9692553fd5515385cb32875e328fd0506965b2afb26e36c6e8e6e
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
/opt # cat /etc/supervisord.conf
[supervisord]
nodaemon=true
user=root
---
docker run --rm -it --entrypoint sh ghcr.io/jorenn92/maintainerr:1.7.1
Unable to find image 'ghcr.io/jorenn92/maintainerr:1.7.1' locally
1.7.1: Pulling from jorenn92/maintainerr
c30352492317: Pull complete
a7eb1ad3dce8: Pull complete
08e4b7445e13: Pull complete
41320be8b6a2: Pull complete
4f4fb700ef54: Pull complete
c59d2624e962: Pull complete
9ad10274a8d2: Pull complete
5fc43397aa34: Pull complete
Digest: sha256:3ad0e51af698c3626dc6d63231e76f65df98bfd1ceaf692a80bac6292ee6313f
Status: Downloaded newer image for ghcr.io/jorenn92/maintainerr:1.7.1
/opt # cat /etc/supervisord.conf
[supervisord]
nodaemon=true
user=root
```
Related: #662

**Comment by jorenn92** [comment]
The non-root pull request has been merged into the main branch but hasn't been released yet. As it's a breaking change, the upcoming release that incorporates this pull request will be version 2.0.

If you're eager to use it immediately, you can utilize the 'ghcr.io/jorenn92/maintainerr:main' image. Keep in mind that this image may break at any time.

**Comment by samip5** [comment]
> The non-root pull request has been merged into the main branch but hasn't been released yet. As it's a breaking change, the upcoming release that incorporates this pull request will be version 2.0.

Technically it was released at some point as 1.7.1, as otherwise the specific sha with it wouldn't work either aka '1.7.1@sha256:0a826f9b33a53e7a241d0b2d8ba2533c6efb978067d149c4f6801a657853ee53'

**Comment by jorenn92** [comment]
> > The non-root pull request has been merged into the main branch but hasn't been released yet. As it's a breaking change, the upcoming release that incorporates this pull request will be version 2.0.
> 
> Technically it was released at some point as 1.7.1, as otherwise the specific sha with it wouldn't work either aka 1.7.1@sha256:0a826f9b33a53e7a241d0b2d8ba2533c6efb978067d149c4f6801a657853ee53 so why's that?

Oh yeah, I switched CI from Jenkins to GitHub Actions. When testing the 'release' pipeline, it accidentally pushed an image of 'main' as version 1.7.1. I quickly fixed it, even before it was mentioned in the documentation that images are available on ghcr.io as well. I didn’t think much if it at the time, but that’s probably the reason.
